exersices/ex01/ex01.rst

```diff
@@ -0,0 +1,143 @@
+Exersice 1
```

Comment:
This file should not be included in the pull request. If you want to include it, commit it in another branch e.g. directly to master.